sdg/open_sdg.py

```diff
@@ -155,7 +155,10 @@ def open_sdg_indicator_options_defaults():
             'GeoCode',
             'Observation status',
             'Unit multiplier',
-            'Unit measure'
+            'Unit measure',
```

Comment:
Unit measure was something we suggested adding, as we added it to our data files in preparation for SDMX with a plan to map it to UNIT_MEASURE. I'm not sure of the best way to deal with this now as we will have 3 separate things for units: Units, Unit measure and UNIT_MEASURE.

Similarly Observation status and Unit multiplier are also common SDMX codes (in fact I think they're required) so not sure if anything needs to be done with those?

Reply:
@LucyGwilliamAdmin The question about OBS_STATUS and UNIT_MULT seems related to our SDMX output project. Our thinking there is that we should be able to produce SDMX even without specifying these required columns in the data. Instead they should be inserted automatically with defaults or configuration at the indicator level.

About the 3 separate units, yes I think this is something to discuss with your team. I see that most indicators seem to have "Unit measure", which is being ignored by Open SDG, but some have "Units", which Open SDG is displaying. Is there any reason that you don't want some indicators to display the unit toggle? (Maybe because they only have one unit?)

Also just to give you a preview of what I'm thinking: All things being equal I would prefer if Open SDG offered _no_ flexibility when it came column names. I think the platform will be more robust and easy to understand when we have clear and definite requirements about column names. And actually the platform is already like that: the column names "Year", "Value", "Series", and "Units" all have specific meanings in Open SDG and can't be spelled any other way. However as we're moving towards Open SDG as a way to input **and output** SDMX, it's become more clear to me that we should have used the SDMX codes, like "TIME_PERIOD" (or whatever), "OBS_VALUE", "SERIES", and "UNIT_MEASURE".

So my long-term goal is to switch over to those SDMX codes. In the short term though, I'm trying to make it optional so that they can be used right now (this is the motivation for this PR: https://github.com/open-sdg/open-sdg/pull/1027). Maybe in Open SDG 2.0.0 or 3.0.0 we can switch to requiring "UNIT_MEASURE" instead of "Units", for example.